assets/dev/js/frontend/handlers/base.js

```diff
@@ -33,7 +33,8 @@ module.exports = elementorModules.ViewModule.extend( {
 		var $mainElement = this.$element;
 
 		return $mainElement.find( selector ).filter( function() {
-			return jQuery( this ).closest( '.elementor-element' ).is( $mainElement );
+			// Start `closest` from parent since self can be `.elementor-element`.
+			return jQuery( this ).parent().closest( '.elementor-element' ).is( $mainElement );
```

Comment:
are you trying to find the "tab" container? if yes can you assign to it some attr?

Reply:
`findElement` does not take in advantage nested elements, and using old code, it will give him wrong handler.